libs/common/src/bindings/c/value.cpp

```diff
@@ -58,8 +58,10 @@ LD_EXPORT(enum LDValueType) LDValue_Type(LDValue val) {
             return LDValueType_Object;
         case Value::Type::kArray:
             return LDValueType_Array;
+        default:
```

Comment:
I feel this has a pretty steep impact on API usage. Is it any better than the null type?

Reply:
Yea, after reflection this is not optimal. We have an underlying `variant` which already has a type tag; let me try taking advantage of that instead with an exhaustive visitor. Maybe the static analysis will be satisfied.